### PR TITLE
System Test - E2E: zli connect

### PR DESCRIPTION
**Pull request by ymarcus93**
## Description of the change

Adds system test for `zli connect`

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] Refactor (code change with no functionality change)
- [ ] Enhancement (minor change below the level of a feature)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Remove feature (removes a feature we don't support anymore)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related JIRA tickets

Relates to JIRA: CWC-1237

## Relevant release note information

Release Notes:

## Potential Security Impacts

Does this PR have any security impact?
Does it fix a security issue?
Does it add attack surface?
Why do we think this change is safe?

### Development

- [ ] Does the code changed/added as part of this pull request have test coverage?
- [ ] Do all tests related to the changed code pass in development?
- [ ] Have you ensured that at least one other person has tested your code?
- [ ] Have you tested the code?
- [ ] Have you attached any pictures (if relevant)?

### CODE REVIEW

File: src/system-tests/tests/suites/list-targets.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { callZli } from '../system-test.test';
+import * as ListTargetsService from '../../../services/list-targets/list-targets.service';
+
+export const listTargetsSuite = () => {
```
Comment (ymarcus93) [inline src/system-tests/tests/suites/list-targets.ts:4]: Do we want to keep this?

File: src/system-tests/tests/suites/version.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { callZli } from '../system-test.test';
+import { version } from '../../../../package.json';
+
+export const versionSuite = () => {
```
Comment (ymarcus93) [inline src/system-tests/tests/suites/version.ts:4]: Do we want to keep this?

File: src/system-tests/tests/suites/list-targets.ts (hunk 1 above)
Comment (ymarcus93) [reply src/system-tests/tests/suites/list-targets.ts:4]: It's not really testing anything

File: src/system-tests/tests/suites/version.ts (hunk 2 above)
Comment (lipmas) [reply src/system-tests/tests/suites/version.ts:4]: yes lets keep this it doesnt cost much in terms of time and its a nice basic test of calling zli

File: src/system-tests/tests/suites/list-targets.ts (hunk 1 above)
Comment (lipmas) [reply src/system-tests/tests/suites/list-targets.ts:4]: agreed we should add some more to this test before we merge the feature branch but lets keep it.

**Review by lipmas — APPROVED** [review]

**Pull request merged**